semgrep-core/src/matching/SubAST_generic.ml

```diff
@@ -160,6 +163,10 @@ let subexprs_of_expr e =
   | DisjExpr _ -> raise Common.Impossible
   [@@profiling]
 
+(* Need this wrapper because [@@profiling] has the side-effect of removing labels. *)
```

Comment:
If you feel like you want to learn ppx and metaprogramming, you can fix it here :) 
https://github.com/returntocorp/pfff/blob/develop/ppx/ppx_profiling.ml

Reply:
I actually want. 😄 I had read some articles about it but never actually wrote any PPX extension. I just add this to my to-dos.